src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/CatalogApiServiceImpl.scala

```diff
@@ -0,0 +1,39 @@
+package it.pagopa.interop.backendforfrontend.api.impl
+
+import akka.http.scaladsl.marshalling.ToEntityMarshaller
+import akka.http.scaladsl.server.Directives.onComplete
+import akka.http.scaladsl.server.Route
+import com.typesafe.scalalogging.{Logger, LoggerTakingImplicit}
+import it.pagopa.interop.backendforfrontend.api.CatalogApiService
+import it.pagopa.interop.backendforfrontend.error.Handlers.handleError
+import it.pagopa.interop.backendforfrontend.model.{EServiceSeed, OldEService, Problem}
+import it.pagopa.interop.backendforfrontend.server.impl.Main.loggerTI.canLogEv
+import it.pagopa.interop.backendforfrontend.service.CatalogProcessService
+import it.pagopa.interop.backendforfrontend.service.types.CatalogProcessServiceTypes.CatalogEServiceSeedConverter
+import it.pagopa.interop.backendforfrontend.service.types.CatalogProcessServiceTypes.CatalogOldEServiceConverter
+import it.pagopa.interop.commons.logging.ContextFieldsToLog
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Success
+
+final case class CatalogApiServiceImpl(catalogProcess: CatalogProcessService)(implicit ec: ExecutionContext)
```

Comment:
in the BFF already exists the `EServicesApiServiceImpl`, you should move this logic there